### ISSUE TITLE
Could I request an API documentation please??

### ISSUE THREAD

**Issue by shawnkoon** [darcikhey27/Spring-Boot-REST-API#1]
I like how this is very active API repo!

I never done Java API layer, so this is new to me which I would love to know more about.

If possible could you post a documentation on ReadMe.md please?

**Comment by darcikhey27** [comment]
yeah, I will do that. Thanks for telling me, I seem to ignore the READme file but it is very important to have documentation about the project.
